app/routes/search/agendaitems.js

```diff
@@ -62,6 +62,10 @@ export default class AgendaitemSearchRoute extends Route {
     if (entry.title && Array.isArray(entry.title)) {
       entry.title = entry.title.join('');
     }
+    if (agendaitem.highlight.shortTitle[0]?.split(" ").length < agendaitem.attributes.shortTitle?.split(" ").length) {
```

Comment:
This doesn't do the proper check, as the `agendaitem.highlight.shortTitle[0]` will still include the ems. To be discussed

Reply:
oh wait, nevermind. It's the length of the split array that's checked here, which will work

Reply:
test is likely failing here when `agendaitem.highlight` is `null`, and/or when `agendaitem.highlight.shortTitle` is falsy (undefined or null). 

Pretty sure `agendaitem.highlight.shortTitle` is always an Array when it's not null, but can't hurt to check either.

@StuyckRuben remember to not always go by the happy flow. Always try to break your own code ;-)